azure-vote/azure-vote/main.py

```diff
@@ -26,16 +26,11 @@
     title = app.config['TITLE']
 
 # Redis configurations
-redis_server = os.environ['REDIS']
+redis_server = 'azure-vote-back'
```

Comment:
One thought in using an environment variable was to make this vote front end portable (used hosted redis, container, local install, etc..). What are your thoughts on hard coding the location here?

Reply:
My idea was since we have the _convention_ in Docker that you can use the service name as the host name, it wasn't necessary to add an env variable to set it by _configuration_.

But I wasn't aware of the use case to run the app outside of docker-compose, since it wasn't a step in the documentation above. 

If running outside of compose is still relevant I can add it back.